### PR TITLE
Add tests for Entity - Default page

### PR DESCRIPTION
**Pull request by OGoncharenko**
https://trello.com/c/HI7cjkGy/935-at11js

### CODE REVIEW

File: test/specs/DefaultViewRecord.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        browser.maximizeWindow();
+        browser.url('/');
+        userLogin(testData.userLogin.loginName, testData.userLogin.password);
+        reset();
+    });
+
+    it('Should create default record with default values', () => {
+        $(selectors.default.sidebarDefault).click();
+        $(selectors.default.addNewRecordButton).click();
+        const saveButton =  $(selectors.default.saveButton);
```
Comment (Burdishka) [inline test/specs/DefaultViewRecord.js:18]: no reason to create this "saveButton" variable
$(selectors.default.saveButton) is enough, it's already clear

File: test/specs/DefaultViewRecord.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        $(selectors.default.addNewRecordButton).click();
+        $(selectors.default.inputString).click();
+        $(selectors.default.inputString).setValue(testData.default.string);
+        $(selectors.default.inputText).click();
+        $(selectors.default.inputText).setValue(testData.default.text);
+        $(selectors.default.inputInt).click();
+        $(selectors.default.inputInt).setValue(testData.default.integer);
+        $(selectors.default.inputDecimal).click();
+        $(selectors.default.inputDecimal).setValue(testData.default.decimal);
+        const saveButton =  $(selectors.default.saveButton);
```
Comment (Burdishka) [inline test/specs/DefaultViewRecord.js:37]: no reason to create this "saveButton" variable
$(selectors.default.saveButton) is enough, it's already clear

File: test/specs/DefaultViewRecord.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    });
+
+    it('Should create default record with default values', () => {
+        $(selectors.default.sidebarDefault).click();
+        $(selectors.default.addNewRecordButton).click();
+        const saveButton =  $(selectors.default.saveButton);
+        saveButton.scrollIntoView();
+        saveButton.click();
+        expect($(selectors.default.firstRowStringCell)
+            .getText().includes('DEFAULT STRING VALUE'))
```
Comment (Burdishka) [inline test/specs/DefaultViewRecord.js:22]: move 'DEFAULT STRING VALUE' to expected.js

File: test/specs/DefaultViewRecord.js [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+        $(selectors.default.inputText).setValue(testData.default.text);
+        $(selectors.default.inputInt).click();
+        $(selectors.default.inputInt).setValue(testData.default.integer);
+        $(selectors.default.inputDecimal).click();
+        $(selectors.default.inputDecimal).setValue(testData.default.decimal);
+        const saveButton =  $(selectors.default.saveButton);
+        saveButton.scrollIntoView();
+        saveButton.click();
+        expect($(selectors.default.secondRowStringCell)
+            .getText().includes('String'))
```
Comment (Burdishka) [inline test/specs/DefaultViewRecord.js:41]: all expected data should be collected in expected.js

File: test/specs/DefaultViewRecord.js [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+            .to.equal(true);
+        expect(getRowCount()).to.equal(expected.common.twoRecords);
+    });
+
+    it('Should edit created record', () => {
+        $(selectors.default.secondRowActionMenu).click();
+        browser.waitUntil(() => {
+            return $("//tr[2]/td[11]/div")
+                .getAttribute("class") === "dropdown pull-left show";
+        });
```
Comment (Burdishka) [inline test/specs/DefaultViewRecord.js:51]: selectors move to selectors.js
expected data move to expected.js

File: test/specs/DefaultViewRecord.js [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+        $(selectors.default.secondRowEditBtn).click();
+        $(selectors.default.inputString).click();
+        $(selectors.default.inputString).setValue(testData.default.editString);
+        const saveButton =  $(selectors.default.saveButton);
```
Comment (Burdishka) [inline test/specs/DefaultViewRecord.js:55]: no reason to create this "saveButton" variable
$(selectors.default.saveButton) is enough, it's already clear

File: test/specs/DefaultViewRecord.js [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+        saveButton.scrollIntoView();
+        saveButton.click();
+        expect($(selectors.default.secondRowStringCell)
+            .getText().includes('String1'))
```
Comment (Burdishka) [inline test/specs/DefaultViewRecord.js:59]: all expected data should be collected in expected.js

File: test/specs/DefaultViewRecord.js [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+            .to.equal(true);
+    });
+
+    it('Should delete edited record', () => {
+        $(selectors.default.secondRowActionMenu).click();
+        browser.waitUntil(() => {
+            return $("//tr[2]/td[11]/div")
+                .getAttribute("class") === "dropdown pull-left show";
```
Comment (Burdishka) [inline test/specs/DefaultViewRecord.js:67]: selectors move to selectors.js
expected data move to expected.js

File: test/specs/DefaultViewRecord.js [hunk 9]
```diff
@@ -0,0 +1,84 @@
...
+        });
+        $(selectors.default.secondRowDeleteBtn).click();
+        browser.pause(500);
```
Comment (Burdishka) [inline test/specs/DefaultViewRecord.js:70]: no pause

File: test/specs/DefaultViewRecord.js [hunk 10]
```diff
@@ -0,0 +1,84 @@
...
+        $(selectors.navigationBar.recycleBinBtn).click();
+        browser.pause(500);
```
Comment (Burdishka) [inline test/specs/DefaultViewRecord.js:72]: no pause

**Review by Burdishka — COMMENTED** [review]
I am sure you don't need change package.json

Also check helper/methods - you will find methods that you can use

**Comment by Burdishka** [comment]
And resolve conflict
You should always check updates from **master** before **push** your **branch**

File: test/specs/DefaultViewRecord.js (hunk 1 above)
Comment (OGoncharenko) [reply test/specs/DefaultViewRecord.js:18]: it didn't work without this const

Comment (Burdishka) [reply test/specs/DefaultViewRecord.js:18]: 🤔 I've just worked with your branch without this const and everything works

Comment (OGoncharenko) [reply test/specs/DefaultViewRecord.js:18]: SaveButton is used two times further in code, so I decided to extract selector into variable.

**Pull request closed without merging**